tests/foreman/smoke/test_cli_smoke.py

```diff
@@ -474,12 +474,14 @@ def test_positive_end_to_end(self):
             {u'organization-id': new_org['id']},
             per_page=False
         )
+        self.assertGreater(len(result), 0)
```

Comment:
For reviewers, the idea here is to fail the test if no results are returned (and avoid getting a traceback further down as is the case right now)